resources/msi/source.wxs.erb

```diff
@@ -12,23 +12,57 @@
           Version="$(var.VersionNumber)" Manufacturer="!(loc.ManufacturerName)" UpgradeCode="$(var.UpgradeCode)">
 
     <!--
-      Define the minimum supported installer version (2.0).
-      The install should be done for the whole machine not just the current user
+      Minimum installer version (5.0) - Window 7/Server 2008 R2 and above.
+      The install scope is per machine, not the current user
     -->
-    <Package InstallerVersion="200" InstallPrivileges="elevated" Compressed="yes" InstallScope="perMachine" />
+    <Package InstallerVersion="500" InstallPrivileges="elevated"
```

Comment:
Not required for fast msi - it's a nice to have/best practice pattern that allows us to set our supported platforms and potentially make use of the latest installer features that come with the updated versions.